src/js/components/Calendar/Calendar.js

```diff
@@ -246,20 +238,17 @@ const Calendar = forwardRef(
     }, [activeDateProp]);
 
     // function that runs inside the useEffect for date and dates
-    const normalizeDate = (dateValue) => {
-      // convert values to UTC based on if date is string or array
+    const normalizeDate = (dateValue, timestamp) => {
```

Comment:
Can we move this function outside of the Calendar component? I don't think it needs any of the Calendar state and can rely solely on the function arguments.

Reply:
Yes, agreed.